.github/workflows/ci.yml

```diff
@@ -125,6 +125,8 @@ jobs:
 
     - run: ./gradlew clean assemble -Dbuild.version_qualifier=${{ env.TEST_QUALIFIER }} && test -s ./build/distributions/opensearch-security-${{ env.SECURITY_PLUGIN_VERSION_ONLY_NUMBER }}-${{ env.TEST_QUALIFIER }}-SNAPSHOT.zip
 
+    - run: ./gradlew clean versionIncrement -DnewVersion=3.0.0-SNAPSHOT
```

Comment:
This executes the command, but doesn't verify any outcome.  

What do you think about adding a execution of the gradle task `assemble`, check the artifact is version X, then run the `versionIncrement` task, then `assemble` and check its version X+1